<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServoWgt</class>
 <widget class="QWidget" name="ServoWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Servo</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>H:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="servohorizontalScrollBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="HorizontalServoAngle">
       <property name="styleSheet">
        <string notr="true">background:green</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>V:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="servoVerticalScrollBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="VerticalServoAngle">
       <property name="styleSheet">
        <string notr="true">background:yellow</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hor_Motor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="HorMotorStateLabel">
       <property name="text">
        <string>stop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ver_Motor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="VerMotorStateLabel">
       <property name="text">
        <string>stop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
